crates/typst/src/layout/transform.rs

```diff
@@ -182,6 +182,14 @@ fn layout_rotate(
 /// ```
 #[elem(Show)]
 pub struct ScaleElem {
+    /// The scaling factor for both axes, as a positional argument. This is just
+    /// an optional shorthand notation for settingq `x` and `y` to the same
+    /// value.
+    #[external]
+    #[positional]
+    #[default(Smart::Custom(ScaleAmount::Ratio(Ratio::one())))]
+    pub factor: Smart<ScaleAmount>,
+
```

Comment:
Wow, this was pretty obscure in the source code. Though it's understandable that it's implemented this way.